docs/source/user-guide/vision/supported_tasks_and_formats.rst

```diff
@@ -0,0 +1,156 @@
+    the data must also be loaded in a **shuffled** manner.
+
+We will first describe the formats which are common for all tasks, and then describe the label and
+prediction formats which vary between tasks.
+
+Common Formats
+==============
+Batch General Format
+--------------------
+Each batch, which is the output of the :doc:`batch loader </user-guide/vision/vision_data_class>`,
+is required to be a :class:`dictionary <deepchecks.vision.vision_data.BatchOutputFormat>`
+with keys for each provided data input: images, labels, predictions and image identifiers.
+Since each data input represents multiple samples,
+all data inputs are expected to be provided
+as either an iterable in which each entry represents a single sample, or as a high dimension array or tensor in
+which the first dimension is the number of samples.
+
+Image Format
+------------
+Each image in the batch can either be a
+`PIL image object <https://pillow.readthedocs.io/en/stable/reference/Image.html>`_
+or a ``[H, W, C]`` 3D array (can be either numpy or a tensor),
+where ``H``, ``W`` and ``C`` represent the height, width and channel of the image.
+The array values should be integers in the range [0, 255]. Color images should be in RGB format and
+have 3 channels, while grayscale images should have 1 channel.
+
+
+Image Identifiers Format
+------------------------
+Image identifiers is a mechanism that allow identifying relevant images in the check results so that they can
+be easily retrieved later on from the original dataset by the user.
+
+Common implementations for this can be the path to the image or the image name or id in a database.
+Image identifiers need to be provided as an iterable of strings.
+
+.. _supported_tasks__classification:
+Classification
+==============
+Label Format
+------------
+Classification label per sample is expected to be provided as either a string or an int
+representing the class index. For example, labels for a batch containing 2 samples can be provided as:
+
+.. code-block:: python
+
+    [1, 3]
+
+Prediction Format
+-----------------
+Classification prediction for each sample should be the **prediction probabilities** per class, meaning that it should be an
+iterable of floats in the range [0, 1] with length equal to the number of classes.
+
+For example, predictions of a batch containing 2 samples for a classification task with 3 possible
+classes may look like this:
+
+.. code-block:: python
+
+    [
+        [0.1, 0.2, 0.7],
+        [0.3, 0.6, 0.1]
+    ]
+
+
+.. _supported_tasks__object_detection:
+Object Detection
+================
+Label Format
+------------
+Object detection label per sample should be an iterable of bounding boxes. Each bounding box should be an iterable
+of 5 elements in the following order: ``(class_id, x_min, y_min, w, h)``.
+
+    ``x_min`` and ``y_min`` are the coordinates (in pixels) of the **top left corner** of the bounding box, ``w``
```

Comment:
are this (and the one below, in the prediction format) indented on purpose?
this is how it looks
![220125 at 12 58 49 screenshot Arc](https://user-images.githubusercontent.com/33841818/214546268-b7b4f231-a8eb-441a-9c0c-5a34b3495412.png)

Reply:
It's the same in label format, and I think it's good

Reply:
Yes mentioned it was the same on both (and put here the printscreen from the other one lol)
just wondered if it was meant to be a "note" or something like that.
I would consider making them bullets, may be clearer/more aesthetical
anyway not critical

Reply:
I think that bullets would be an overkill here.